extended range version.js

```diff
@@ -1,86 +0,0 @@
-const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
-
-start();
-
-
-
-async function start() {
-
-
-
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let userMax = await ask("First, let's set the range. We'll play between 1 and a number of your choosing. Please enter the max limit now. ")
-  console.log("You entered: " + userMax)
-  let secretNumber = await ask("Now pick a number between 1 and the number you just entered - What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-
-  let min = 1
-  let max = userMax
-
-  //something about the way this is running is giving numbers way outside the specified range
-  function randNum(min, userMax) {
-    let range = (userMax - min) + 1;
-    let decimal = (Math.random() * range);
-    let int = Math.floor(decimal) + min;
-    return int;
-  }
-
-
-
-  let compGuess = randNum(min, max)
-  //if i can limit the number of repitions (or asks), that could work as a cheat guard
-
-  answer = await ask(`Is your number ${compGuess}? Please enter 'yes' or 'no': `)
-
-  while (answer !== 'yes' && answer !== 'no') {
-    answer = await ask("Please enter either 'yes' or 'no': ")
-  }
-
-  if (answer === 'no') {
-
-    while (answer === 'no') {
-      console.log("inside the while loop!")
```

Comment:
when you have use `console.log` statements like this to check that you're where you expect to be. It's good practice to remove them once they have served their purpose.

It's very easy to leave them and before you know it, your console ends up quite full of statements being printed and it's easy to get lost with what you're looking for.